web/concrete/src/Application/Application.php

```diff
@@ -130,6 +130,45 @@ public function clearCaches()
         OpCache::clear();
     }
 
+   /**
+     * Utility method for clearing page caches and block caches that has btCacheBlockOutputClearOnEvents option enabled
+     */
+    public function partialClearCaches()
+    {
+        \Events::dispatch('on_partial_cache_flush');
```

Comment:
Can we use the objects on the container instead of calling the facade everywhere in here? `$this['director']->dispatch(...)` and $this['database']`